### ISSUE TITLE
Blocks flickering/transparent edges with shaders [1.14.3]

### ISSUE THREAD

**Issue by minecraftvalley** [sp614x/optifine#2628]
I have run Optifine with SEUS shaders  (SEUS Renewed 1.0.0) on the previous build (1.13.2) with no problems, so I believe this issue arises with either SEUS itself, or more likely, it's compatibility with Optifine 1.14.3.

The game starts fine, then randomly (pattern seems to be around 10-15 minutes of play, but not always) blocks will start bouncing less than a block to the side, then back, showing transparency through the blocks. This has happened on both singleplayer worlds and multiplayer servers/realms, so it's 100% client-side. It only happens when shaders are enabled, which is the reason I downloaded Optifine in the first place, so disabling shaders isn't really an option for me.

Per the suggestion of a few others in the server, I tried turning fast-render off, and changing mipmap levels to 2. Both were to no avail.The only option that I'd toggled that seemed to actively be changing how fast the flickering occurred was the rendering quality slider. Raising the quality raised the speed of the flickering, until messing with it enough cycling through the speeds just stopped the bug altogether.

I am running minecraft on my computer that uses an Nvidia graphics card, which may be a common issue with users running into this bug.

I'm not positive this bug is caused by SEUS alone, so I'm posting this here as recommended by other players. I'm willing to share info on my other settings if necessary.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/52928825/61233175-90ea8e00-a6f5-11e9-8feb-e86a8977a090.gif)

**Comment by sp614x** [comment]
Looks like a time accumulation bug in SEUS.
Does it always happen at the same time (10-15 min) after game start?

**Comment by minecraftvalley** [comment]
I'm not sure. It seems to follow that pattern but I'll have to start keeping better track of it to be sure. I've temporarily fixed it for now by turning on Render Regions in OptiFine, but sometimes it starts up again.

After the temporary fix, sometimes the bug changes into white dots that blink, and on rare occasions will start behaving like previously, but only in the distance. I'll look into asking for SEUS support though and report back if it happens at random intervals instead of around the same time, thank you.

**Comment by gurnben** [comment]
@minecraftvalley I see that this issue is still open, have we seen a resolution from SEUS?  I'm encountering the same issue with SEUS Renewed on 1.14.4 currently.  

In my case the error occurred immediately after opening the game and persisted through 2 restarts.

**Comment by Verifirs** [comment]
I had this issue too, I solved it by turning _off_ render regions.

**Comment by gurnben** [comment]
SEUS Renewed 1.0.1 seems to have resolved this issue in my experience @Verifirs, are you still seeing it on 1.0.1?  If so that's very interesting and I can see what effect turning off render regions will do in my case.  I assumed that SEUS Renewed 1.0.1 addressed the time accumulation issue sp614x mentioned above!

**Comment by Verifirs** [comment]
Yes, I was seeing this issue in SEUS Renewed 1.0.1 with render regions turned on.
